extension/src/popup/components/sendPayment/SendSettings/Settings/index.tsx

```diff
@@ -109,8 +130,10 @@ export const Settings = ({
       return;
     }
 
-    if (isToken) {
-      const assetAddress = asset.split(":")[1];
+    if (isToken || isSendSacToContract) {
```

Comment:
Sending a token to a C address generally worked "out of the box" once I allowed a user to enter a C address as a destination 🎉